docs/arch-basic.rst

```diff
@@ -0,0 +1,42 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Basic airflow architecture
+==========================
+
+Primarily intended for development use, the basic airflow architecture with the Local and Sequential executors is an excellent starting point for understanding the architecture of Apache Airflow.
+
+.. image:: img/arch-diag-basic.png
```

Comment:
Question. How do we edit this diagram if anything changes? I believe we should have some way to reproduce the work by any member of the community, so that we can make modification to the image.

Reply:
Ya I was meaning to get in touch with you regarding this but went out for dinner in a restaurant :D -- anyways what I was going to say is we will upload all the architecture diagrams to https://cwiki.apache.org/confluence/display/AIRFLOW/File+lists so anyone would be able to download and recreate it easily.